src/state/atoms.ts

```diff
@@ -1,17 +1,15 @@
 import dayjs from "dayjs";
+import isToday from "dayjs/plugin/isToday";
 import { atom, selector, useRecoilState } from "recoil";
 import { Task } from "../types/task";
 
+dayjs.extend(isToday);
```

Comment:
I would add a file in services (e.g. timeUtils.ts) and export extended dayjs object from there, so you don't need to do it everywhere you want to use it.

Reply:
Once I export the object, how can I call is in another file? I am having trouble with this export.

Reply:
How do I export dayjs.extend(isToday)? I have tried several ways and none work.